src/router/storeSync/CrossHairParamConfig.class.js

```diff
@@ -0,0 +1,53 @@
+import AbstractParamConfig from '@/router/storeSync/abstractParamConfig.class'
+import { round } from '@/utils/numberUtils'
+
+function dispatchCrossHairFromUrlIntoStore(store, urlParamValue) {
+    const promisesForAllDispatch = []
+
+    const parts = urlParamValue.split(',')
+    if (parts.length === 1) {
+        promisesForAllDispatch.push(store.dispatch('setCrossHair', { crossHair: urlParamValue }))
+    } else if (parts.length === 3) {
+        const crossHair = parts[0]
+        const crossHairPosition = [parseFloat(parts[1]), parseFloat(parts[2])]
+        promisesForAllDispatch.push(
+            store.dispatch('setCrossHair', { crossHair, crossHairPosition })
+        )
+    }
```

Comment:
I'm not sure this is a good idea to add the crosshair position into the crosshair parameter. At least my understanding of #451 this would not be needed as all they would need is to set the `center` parameter. All what we need here is to reset the crosshair position when the center changes. I tested it with this PR and it works.
Having the position in the crosshair make the application a bit more difficult to understand and could reveal some issue in the future as the behavior of the crosshair is then different when it has the position or not.

Reply:
After a talk with @pakb saving the position with the crosshair is actually a good idea. It give the possibility to move the map around keeping the crosshair at its initial position. Note that on the old viewer the crosshair did not stayed permanently in url so when reloading it was lost, this pr as the advantage to keep it.